app/views/krikri/shared/_navigation.html.erb

```diff
@@ -1,10 +1,45 @@
-<div id="header-navbar" class="navbar navbar-inverse navbar-static-top" role="navigation">
-  <div class="container">
-    <div class="navbar-right">
-    <ul class="nav navbar-nav">
-        <li><%= link_to 'Institutions', krikri.institutions_path %></li>
-        <li><%= link_to 'QA', krikri.records_path %></li>
-    </ul>
+<div id='header-navbar' class='navbar navbar-inverse navbar-static-top' role='navigation'>
+  <div class='container'>
+    <div class='navbar-right'>
+      <ul class='nav navbar-nav'>
+        <li class='dropdown'>
+          <a href='#' class='dropdown-toggle' data-toggle='dropdown'
+            role='button' aria-expanded='false'>
+            <%= provider_name(session[:current_provider]) %> <span class="caret"></span>
+          </a>
+          <ul class='dropdown-menu' role='menu'>
+	    <li>
+              <%= remove_session_provider %>
+            </li>
+            <% if session[:current_provider].present? %>
+              <li>
+                <%= set_session_provider(session[:current_provider]) %>
+              </li>
+              <li class='divider'></li>
```

Comment:
Do we want to just move it up to underneath the `remove_session_provider` link?

Would this menu be improved by dropping the link to the current provider altogether?